client/blocks/app-promo/index.jsx

```diff
@@ -148,29 +157,31 @@ export class AppPromo extends Component {
 		const { translate } = this.props;
 		const buttons = [ { action: 'cancel', label: translate( 'OK' ) } ];
 
+		const message = displayJetpackAppBranding
+			? translate( '{{a}}Get the Jetpack app{{/a}} to use Reader anywhere, any time.', {
+					components: {
+						a: <a href="https://wp.com/app" className="app-promo__jetpack-app-link" />,
+					},
+			  } )
+			: translate( 'WordPress.com in the palm of your hands — download the mobile app.' );
+
 		return (
 			<div className="app-promo">
 				<Button
-					tabIndex="0"
-					className="app-promo__dismiss"
-					onClick={ this.dismiss }
-					aria-label={ translate( 'Dismiss' ) }
-				>
-					<Gridicon icon="cross" size={ 24 } />
-				</Button>
-				<Button
-					onClick={ this.sendMagicLink }
-					className="app-promo__link"
+					onClick={ ! displayJetpackAppBranding ? this.sendMagicLink : undefined }
+					className={ classNames( 'app-promo__link', {
+						'app-promo_jetpack-app-button': displayJetpackAppBranding,
+					} ) }
 					title="Try the mobile app!"
 				>
 					<img
 						className="app-promo__icon"
-						src={ wordpressLogoImage }
-						width="32"
-						height="32"
+						src={ displayJetpackAppBranding ? jetpackLogoImage : wordpressLogoImage }
+						width={ displayJetpackAppBranding ? 25 : 32 }
+						height={ displayJetpackAppBranding ? 43 : 32 }
 						alt="WordPress App Icon"
```

Comment:
Similar to a comment on the desktop app `alt` text, I wonder how beneficial this `alt` value is. WDYT about using an empty string instead? 

If we do keep `alt` text here, we need to dynamically mention "Jetpack" instead of "WordPress".

Reply:
Addressed as part of [this comment](https://github.com/Automattic/wp-calypso/pull/68293#discussion_r988345981). 🙇‍♀️